src/ocm/contexts/FeatureSupportLevelContextProvider.tsx

```diff
@@ -0,0 +1,137 @@
+import useSWR from 'swr';
+import React, { PropsWithChildren } from 'react';
+import FeatureSupportLevelContext, {
+  SupportLevelDataInterface,
+  EmptySupportLevelData,
+} from '../../common/contexts/FeatureSupportLevelContext';
+import { ClusterFeatureUsage, FeatureId, SupportLevel, SupportLevelMap } from '../../common/types';
+import { getErrorMessage, handleApiError } from '../api/utils';
+import { FeatureSupportLevels, useAlerts } from '../../common';
+import { FeatureSupportLevelsAPI } from '../services/api';
+
+class FeatureSupportLevelData implements SupportLevelDataInterface {
```

Comment:
```suggestion
class FeatureSupportLevelData implements SupportLevelDataInterface {
```
It's long enough to be worth having its own file, wdyt?

Reply:
Thanks for the reference, I'll try to use it in a separate PR